src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -30,1327 +31,990 @@
 import net.minecraft.world.item.*;
 import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.Blocks;
-import net.minecraftforge.event.RegistryEvent;
-import net.minecraftforge.eventbus.api.SubscribeEvent;
 import net.minecraftforge.fml.common.Mod;
-import net.minecraftforge.registries.IForgeRegistry;
-import net.minecraftforge.registries.ObjectHolder;
+import net.minecraftforge.registries.*;
 
-import javax.annotation.Nonnull;
+import java.util.function.Function;
 
 import static com.mraof.minestuck.block.MSBlocks.*;
 
 /**
- * This class contains all non-ItemBlock items that minestuck adds,
- * and is responsible for initializing and registering these.
+ * This class initializes and registers all items and blockitems in the mod. Utilizes a DeferredRegister, so utilizations of items from this class will often require .get()
  */
-@ObjectHolder(Minestuck.MOD_ID)
 @Mod.EventBusSubscriber(modid = Minestuck.MOD_ID, bus = Mod.EventBusSubscriber.Bus.MOD)
 public class MSItems
 {
+	//TODO removed @ObjectHolder(Minestuck.MOD_ID), ensure that was okay to do
```

Comment:
It should be fine, as all fields are now initialized with registry objects from the deferred register.

Reply:
Deleted now. However there was a near identical TODO i made for MSBlocks. Should that one be removed?

Reply:
Yeah